components/generic/SimilarItems.stories.js

```diff
@@ -0,0 +1,48 @@
+import { storiesOf } from '@storybook/vue';
+import StoryRouter from 'storybook-vue-router';
+import SimilarItems from './SimilarItems.vue';
+
+const i18n = {
+  locale: 'en',
+  messages: {
+    en: {}
+  }
+};
+
+storiesOf('Generic', module)
+  .addDecorator(StoryRouter())
```

Comment:
Pass this a named route to prevent console warnings:
```js
  .addDecorator(StoryRouter({}, {
    routes: [
      { name: 'record-all', path: '/record/*' }
    ]
  }))
```